bucket_manifest/utils.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import random
+from datetime import datetime
+import string
+import logging
+import boto3
+from botocore.exceptions import ClientError
+
+
+def randomString(stringLength=10):
```

Comment:
```suggestion
def random_string(string_length=10):
```
lower_case naming convention for python code. Can you make sure to run Black on all this if you haven't already?

Reply:
I have just remove the function since it is not used.